app/code/Magento/MediaStorage/Model/File/Validator/NotProtectedExtension.php

```diff
@@ -78,6 +78,9 @@ protected function _initProtectedFileExtensions()
     {
         if (!$this->_protectedFileExtensions) {
             $extensions = $this->getProtectedFileExtensions();
+            if ($extensions === null) {
```

Comment:
is_null is not allowed. And there for we have multi programming styles now. The Magento style is not allowed for me.